docs/guides/server/importExport.adoc

```diff
@@ -1,12 +1,25 @@
 <#import "/templates/guide.adoc" as tmpl>
 <#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
 
 <@tmpl.guide
     title="Importing and Exporting Realms"
     summary="An overview about how to import and export realms">
 
 In this guide, you are going to understand the different approaches for importing and exporting realms using JSON files.
 
+== Providing options for database connection parameters
+
+When using the `export` and the `import` commands below, Keycloak needs to know how to connect to the database where the information about realms, clients, users and other entities is stored.
```

Comment:
I'm not sure if we need a section to explain how to use build options (E.g.: database).

As part of the introduction, we could :

```
The `export` and `import` commands work very similarly 
to the `start` command except that it won't accept any 
incoming HTTP connection and once the export or import
operation is complete, the server will stop automatically. 
You are also able to use both build options and configuration 
options as per  <@links.server id="configuration"/>.
```